.gitignore

```diff
@@ -22,6 +22,7 @@ node_modules/
 # Generated files
 *.abi
 abi/
+artifacts/
```

Comment:
I'm not sure if `Generated files` is a good section for this directory, it feels more like `Build output`, or even `Deployment output`.